scripts/validateBackupDB.sh

```diff
@@ -15,36 +15,17 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 # [MUST BE FILLED IN] User configuration settings
-aws_access_key_id=""
-aws_secret_access_key=""
-
+backupDBName=""
 
 # Validate input argument
-if [ "$#" -ne 1 ]; then
+if [ ! -f "${backupDBName}" ]; then
     echo -e "\n\n\033[0;31mUsage: source ./scripts/validateBackupDB.sh backupDBName\033[0m\n"
-    return 0
-fi
-
-if [ "$aws_access_key_id" = "" ]; then
-    echo -e "\n\n\033[0;31m*ERROR* Please enter your own AWS_ACCESS_KEY_ID in validateBackupDB.sh!\033[0m\n"
-    return 0
+    return 1
 fi
 
-if [ "$aws_secret_access_key" = "" ]; then
-    echo -e "\n\n\033[0;31m*ERROR* Please enter your own AWS_SECRET_ACCESS_KEY in validateBackupDB.sh!\033[0m\n"
-    return 0
-fi
-
-
 # Download persistence from Amazon S3 database
-backupDBName="$1"
-export AWS_ACCESS_KEY_ID=${aws_access_key_id}
-export AWS_SECRET_ACCESS_KEY=${aws_secret_access_key}
-pip install awscli
-aws s3 cp s3://zilliqa-persistence/${backupDBName}.tar.gz ${backupDBName}.tar.gz
 rm -rf persistence
-tar xzvf ${backupDBName}.tar.gz
-echo -e "\n\033[0;32mDownload ${backupDBName}.tar.gz from Amazon S3 database successfully.\033[0m\n"
```

Comment:
Why remove logic of downloading from s3. Is it moved out ??

Reply:
The validation should focus on the database correctness, and the removal of S3-related logic could give this script more flexibility. That is, once user have a persistence in hand, he can use this script immediately without uploading the database to S3 first.